Src/Plugins/SalModernization/SalModernization.csproj

```diff
@@ -11,6 +11,12 @@
     <PackageId>Sarif.PatternMatcher.SalModernization</PackageId>
   </PropertyGroup>
 
+  <PropertyGroup>
+	<MSBuildWarningsAsMessages>
+	    $(MSBuildWarningsAsMessages);MSB3277;MSB3030
+	</MSBuildWarningsAsMessages>
+  </PropertyGroup>
```

Comment:
@schlaman-ms There seems to be code duplication. Can we move this to build.common.props perhaps?

Reply:
Well I intentionally only put the MSBuildWarningsAsMessages entries in these projects as the suppressions only occur in the project where the entry is.

If I put this in build.common.props then I would suppress those errors globally and that wouldn't be good.
There is already a certain amount of risk by suppressing that you silence something that shouldn't be silenced. If in a given project, I only need to silence one warning then that project only gets the one MSBxxxx entry.

Reply:
@schlaman-ms To me, the problems of the duplication is showing up already. If you make a refinement to the XML comment, that adds a follow-on work item to make sure it's updated everywhere. The `build.common.props` was just a suggestion. Can we not create a new suppression file, and make the relevant projects reference that instead? Do we currently have a scenario where we are suppressing things other than MSB3277 and MSB3030?

Reply:
I agree with it is better if possible, to define in one place, even if it applies to more projects those not have to have it.
applies to more projects has not have major impact.
The benefit is 
less duplicate code, 
easier to maintain, 
and if the project that currently does not need starts to have the e.g. newton in the future then it is already covered no need to add one by one again.